tuxemon/db.py

```diff
@@ -218,6 +218,8 @@ class MonsterModel(BaseModel):
     slug: str = Field(..., description="The slug of the monster")
     category: str = Field(..., description="The category of monster")
     ai: str = Field(..., description="The AI to use for this monster")
+    txmn_id: float = Field(..., description="The id of the monster")
```

Comment:
I'd should be an int. Height and weight are currently ints, but may as well allow them to be floats.

Reply:
If I change that into an int, then I guess I need to update monster.py as well?

from self.txmn_id = 0.0 to self.txmn_id = 0

I ask so I can avoid pointless fixing (risk of a pointless commit).

Reply:
Yes